.github/docker-images/raspbian-bullseye/Dockerfile

```diff
@@ -0,0 +1,61 @@
+FROM raspbian/stretch as raspbian-bullseye
+
+###############################################################################
+# Upgrade to buster from stretch
+###############################################################################
+
+RUN apt-get update \
+	&& apt-get upgrade -y \
+	&& apt-get full-upgrade -y \
+	&& apt-get autoremove --purge -y \
+	&& apt-get clean -y \
+	# Switch to Buster repository.
+	&& sed -i 's/stretch/bullseye/g' /etc/apt/sources.list \
```

Comment:
multiple comments mention "buster" but code upgrades to "bullseye"

Reply:
Thanks!!